GoogleApisClient.sln

```diff
@@ -48,6 +48,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GoogleApis.WP81", "Src\Goog
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GoogleApis.Auth.WP81", "Src\GoogleApis.Auth.WP81\GoogleApis.Auth.WP81.csproj", "{90660B22-B933-44A6-B34D-E4F4352D4434}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GoogleApis.Auth.DotNet4.Tests", "Src\GoogleApis.Auth.DotNet4.Tests\GoogleApis.Auth.DotNet4.Tests.csproj", "{5A731729-8E99-4930-AE77-FA3524673DA3}"

```

Comment:
I can't see it here, but did you add this project under "DotNet4" (00A1D580-DF06-4C96-A4DA-EDDEF8584971) folder in VS?